litellm/router.py

```diff
@@ -2569,6 +2569,7 @@ async def _pass_through_moderation_endpoint_factory(
                 model=kwargs["model"]
             )
             kwargs["model"] = deployment["litellm_params"]["model"]
+            kwargs["api_key"] = deployment["litellm_params"]["api_key"]
```

Comment:
i don't believe api_key is a guaranteed field of `litellm_params` did this pass your lint check?

Reply:
@krrishdholakia lint check via `poetry run flake8` didn't return error
if you think that param is not guaranteed, how about implementing like below?

```
    if deployment["litellm_params"].get("api_key") is not None:
        kwargs["api_key"] = deployment["litellm_params"]["api_key"]
```